magnolia-voting-app/src/main/resources/voting-contentApp/apps/voting-app.yaml

```diff
@@ -24,6 +24,8 @@ subApps:
         availability:
           writePermissionRequired: true
           rules:
+            notStarted:
```

Comment:
We should not expose details of implementation, so I would change it to pollEditable. User isn't know anything behind jcrIsPollEditableRule or not sure if the rule of editing poll may change.